src/Aspire.Hosting/Dcp/DcpExecutor.cs

```diff
@@ -24,22 +26,35 @@
 using k8s.Autorest;
 using k8s.Models;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Polly;
 
 namespace Aspire.Hosting.Dcp;
 
-internal sealed class DcpExecutor : IDcpExecutor, IConsoleLogsService, IAsyncDisposable
+internal sealed partial class DcpExecutor : IDcpExecutor, IConsoleLogsService, IAsyncDisposable
```

Comment:
I think these changes are not intended.

Reply:
Merged with `main`. Should be fixed now.